cli/command/manifest/util.go

```diff
@@ -0,0 +1,44 @@
+package manifest
+
+import "fmt"
+
+// list of valid os/arch values (see "Optional Environment Variables" section
+// of https://golang.org/doc/install/source
+// Added linux/s390x as we know System z support already exists
+
+var validOSArch = map[string]bool{
+	"darwin/386":      true,
```

Comment:
Hey @stevvooe ! Anything ugly in this PR is probably a copy over from my external "manifest-tool" project :) In this case, I was trying to validate the input against a known universe of Golang os+arch pairs just to sanitize what people might claim as valid pairs. The format itself is just a slash-separated copy of the [Golang list of accepted pairs](https://golang.org/doc/install/source#environment) from golang.org. Of course this in itself is also broad because the Docker engine does not exist for every pair in that list.